whois-api/src/main/java/net/ripe/db/whois/api/nrtm4/WhoisNrtmServletDeployer.java

```diff
@@ -22,9 +25,15 @@ public WhoisNrtmServletDeployer(final NrtmClientService nrtmClientService, final
 
     @Override
     public void deploy(final WebAppContext context) {
+        final JacksonJsonProvider jacksonJsonProvider = new JacksonJsonProvider();
+        jacksonJsonProvider.configure(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, true);
+        jacksonJsonProvider.configure(SerializationFeature.INDENT_OUTPUT, true);
+
+
         final ResourceConfig resourceConfig = new ResourceConfig();
         resourceConfig.register(nrtmClientService);
         resourceConfig.register(nrtmExceptionMapper);
+        resourceConfig.register(jacksonJsonProvider);
```

Comment:
Is adding a `jacksonJsonProvider` (especially the INDENT_OUTPUT setting) causing the entire snapshot JSON to be re-formatted? This may cause serialisation to take a lot more memory and take longer.

Reply:
Or, is there *already* a default `jacksonJsonProvider`

Reply:
I did for some testing forgot to remove. I will remove this